### ISSUE TITLE
About the test results

### ISSUE THREAD

**Issue by Bigtuo** [xincoder/GRIP#26]
I'm sorry to bother you again.
As follows,
What do the seven columns represent in these result data?
Is ADE and FDE displayed?
The evaluation results of an Epoch are as follows,
#######################################Test
|2021-07-09 17:58:51.802929|[car] All_All: 0.449 0.773 1.140 1.520 1.927 2.356 8.165
|2021-07-09 17:58:51.802929|[human] All_All: 0.145 0.267 0.403 0.554 0.715 0.891 2.977
|2021-07-09 17:58:51.802929|[bike] All_All: 0.432 0.826 1.207 1.615 2.009 2.440 8.528
|2021-07-09 17:58:51.802929|[WS] All_All: 0.269 0.491 0.727 0.981 1.242 1.525 5.236
|2021-07-09 17:58:51.818551|[Test_Epoch20] All_All: 0.275 0.498 0.739 1.011 1.289 1.584 5.397

**Comment by xincoder** [comment]
@Bigtuo. The first 6 numbers are ADE (not exactly implemented the metric as the website/paper description, considering the numerical stability problem), and the last number is the sum of the first 6 numbers. During training, one can use these numbers to roughly get some sense about the performance of the model, they are not the correct errors. Please submit your results to the website to get the final error. Thanks.

**Comment by Bigtuo** [comment]
ok,thankyou



---Original---
From: ***@***.***&gt;
Date: Mon, Jul 12, 2021 18:07 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [xincoder/GRIP] About the test results (#26)




 
@Bigtuo. The first 6 numbers are ADE (not exactly implemented the metric as the website/paper description, considering the numerical stability problem), and the last number is the sum of the first 6 numbers. During training, one can use these numbers to roughly get some sense about the performance of the model, they are not the correct errors. Please submit your results to the website to get the final error. Thanks.

**Comment by han1222** [comment]
Is this 6 number means predicted horizon? for example 1 second , 2 second .... 6 second  ?
and what WS does that mean?

Thank you.

**Comment by Bigtuo** [comment]
> 这个 6 数字是指预测的地平线吗？例如 1 秒，2 秒 .... 6 秒？
> WS 是什么意思？
> 
> 谢谢你。

The first 6 are what you said, WS is the sum of the weighted values of car, bike, and person (WSADE = Dv · ADEv + Dp · ADEp + Db · ADEb). This value is mentioned in the paper

**Comment by han1222** [comment]
Thank you.

[Test_Epoch20] All_All: 0.275 0.498 0.739 1.011 1.289 1.584 5.397 
and then what does these values mean?

**Comment by xincoder** [comment]
> Thank you.
> 
> [Test_Epoch20] All_All: 0.275 0.498 0.739 1.011 1.289 1.584 5.397
> and then what does these values mean?

Hi @han1222, thank you for your interest in our work. Those values mean nothing but the performance of the model tested on my own split validation set (80% for training, and 20% for testing, split in xin_feeder_baidu.py [line33-34]). Next, you may want to generate predicted results on the testing set (main.py [line358-360]), and then submit the results to the Apolloscape website to get the final testing results (these numbers are results reported in papers).

**Comment by han1222** [comment]
I really appreciate your kind answer.  Thank you.

**Comment by jasonY18** [comment]
Hello, I would like to inquire about the specific meaning of the 7 numbers in the testing results. Do the first six numbers represent the ADE values for the next three seconds, with two frames per second?
[Test_Epoch49] All_All: 0.267 0.481 0.713 0.975 1.247 1.531 5.214
